package/MDAnalysis/core/AtomGroup.py

```diff
@@ -3336,6 +3378,7 @@ def __init__(self, name, id, atoms, resnum=None):
         ##if not Residue._cache.has_key(name):
         ##    Residue._cache[name] = dict([(a.name, i) for i, a in enumerate(self._atoms)])
 
+    @deprecate(message=_FIFTEEN_DEPRECATION)
```

Comment:
I think these things are really fragile. They depend on particular atom names (however customary), and the implementation also seems a bit frail, depending on residue id being in order for the chain. I'd rather mark them as deprecated and then reverse the decision later if we think they can be included with confidence.

They've been included for now among the ``Atomnames`` TopologyAttr in the 363 branch (dbe6d061e35f45104317e2c3662601f19b9ea44e), but they don't work at the moment since they depend on Residue ``__getitem__`` working with atom names in a funny way. This isn't implemented yet, and I'm not sure it should be, but it is possible. See [``__getattr__`` for Residues](https://github.com/MDAnalysis/mdanalysis/blob/dbe6d061e35f45104317e2c3662601f19b9ea44e/package/MDAnalysis/core/topologyattrs.py#L278) for an example of how.